managed/testdata/pg/conf/pg_hba.conf

```diff
@@ -1,5 +1,9 @@
 local     all         all                               trust
+# IPv4 local connections:
+host      all         all         127.0.0.1/32          trust
+# IPv6 local connections:
+host      all         all         ::1/128               trust
```

Comment:
But what actually changed? Why it worked without these settings before?

Reply:
The failure was due to flaky environment config issues on Github side. It seems, they are OK now, but the fix is not harmful, but helpful in case we face the same situation again.

Reply:
Anyways, I provided the reference to a workflow run, that confirms that it actually was an issue.

Reply:
pmm-agent tests don't use pmm-managed DB, so it does nothing for pmm-agent tests